lib/faker/default/number.rb

```diff
@@ -139,6 +139,21 @@ def hexadecimal(legacy_digits = NOT_GIVEN, digits: 6)
         hex
       end
 
+      # Produces a number in binary format.
+      #
+      # @param digits [Integer] Number of digits in the he
+      # @return [String]
+      #
+      # @example
+      #   Faker::Number.binary(digits: 4) #=> "1001"
+      #
+      # @faker.version 1.0.0
```

Comment:
You should use `next` as version for new methods :)
```suggestion
      # @faker.version next
```